src/models/tenant.cr

```diff
@@ -68,6 +68,7 @@ class Tenant
     validate_columns
     validate_domain_uniqueness
     validate_credentials_for_platform
+    validate_booking_limits
```

Comment:
This could go earlier in the validation to prevent unnecessary IO for an invalid object, perhaps before `validate_domain_uniqueness`

Reply:
Sure, I will move it.